tests/console/acpi.pm

```diff
@@ -0,0 +1,27 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 Bernhard M. Wiedemann
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "consoletest";
+use strict;
+use testapi;
+
+# poo#11798
+sub run() {
+    select_console 'root-console';
+
+    if (get_var("EXTRABOOTPARAMS") =~ /acpi=force/) {
+        assert_script_run "[ -d /sys/firmware/acpi ]";
+    }
+    else {
+        assert_script_run "[ -d /sys/firmware/devicetree ]";
```

Comment:
Fate#319981: (mmarek) Also, it would be good to verify that without acpi=force, we are defaulting to device tree.